docs: add "The xonsh shell community" to the readme (#4585)
xonsh/xonsh: README.rst

- `Regro autotick bot <https://github.com/regro/cf-scripts>`_: Regro Conda-Forge autoticker.
- `xxh <https://github.com/xxh/xxh>`_: Using xonsh wherever you go through the ssh.

The xonsh shell community
*************************

> The xonsh shell is developing by the community of volunteers and have no organisation that can get grants, donations or additional support. There are two major ways to help the xonsh shell: to spread a good word about xonsh shell in the world (`write a tweet`_) and give an effort to improve the xonsh shell (`solve the issue <https://github.com/xonsh/xonsh/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc>`_). We're very welcome the new contributors!

Reviewer: ```suggestion
The xonsh shell is developed by a community of volunteers and has no organization that can get grants, donations or additional support. There are two major ways to help the xonsh shell: to spread the good word about xonsh in the world (`write a tweet`_) and to help improve the xonsh shell (`solve an issue <https://github.com/xonsh/xonsh/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc>`_). We welcome new contributors!
```

Thanks @anki-code !  Some small language tweaks here and shifted a few words to American english spelling (not that it's right or wrong, just to maintain consistency with the rest of the docs)

Author: Thanks!
